toggle-dummy.js

```diff
@@ -1,8 +1,32 @@
-  $(function(){
-    var button =
-      $('<button type="button" onclick="toggleDummy()"> Show dummies</button>');
-    $("body").prepend(button);
-  });
-  function toggleDummy(){
-    $(".dummy").toggleClass("is-active")
+$(function(){
+  addButton()
+  if (exposingDummyElements()){
+    toggleActiveElement();
   }
+});
+
+function addButton() {
+  var button = $('<button type="button" onclick="toggleDummyElements()"> Show dummies</button>');
+  $("body").prepend(button);
+}
+
+function toggleDummyElements(){
+  toggleActiveElement();
+  toggleStorage();
+}
+
+function toggleActiveElement() {
+  $(".dummy").toggleClass("is-active");
+}
+
+function toggleStorage() {
+  if (exposingDummyElements()) {
+    localStorage.setItem('showDummy', false);
```

Comment:
What do you think about putting these in as strings since that is what is going to be stored anyway? That would make the line below a bit less confusing.